### ISSUE TITLE
Undefined reference during make related to Linking CXX executable storage_test

### ISSUE THREAD

**Issue by kochie** [facebook/thpp#37]
So I've run into this error during compilation, not really sure where to begin - If anybody has any ideas that would be great.

`[ 73%] Linking CXX executable storage_test
../libthpp.so: undefined reference to `thpp::_ThriftTensorEndiannessEnumDataStorage::names'
../libthpp.so: undefined reference to `thpp::_ThriftTensorDataTypeEnumDataStorage::values'
../libthpp.so: undefined reference to `thpp::_ThriftTensorEndiannessEnumDataStorage::values'
/usr/lib/gcc/x86_64-pc-linux-gnu/6.3.1/../../../../lib/libthriftcpp2.so: undefined reference to `apache::thrift::transport::THeader::setHeader(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&&)'
../libthpp.so: undefined reference to `thpp::_ThriftTensorDataTypeEnumDataStorage::names'
collect2: error: ld returned 1 exit status
make[2]: *** [test/CMakeFiles/storage_test.dir/build.make:106: test/storage_test] Error 1
make[1]: *** [CMakeFiles/Makefile2:223: test/CMakeFiles/storage_test.dir/all] Error 2
make: *** [Makefile:139: all] Error 2`

**Comment by stalagmite7** [comment]
Hi, I have the same error, have you figured a way out of yours?

**Comment by kochie** [comment]
Yeah if I remember correctly there was something wrong with the repo at this stage, reverting to a previous commit fixed it. I was running on Arch when this happened and I think I gave up to be honest. I'll have another go and see if anything changes.
